### PR TITLE
Expose active observation in python

### PR DESCRIPTION
**Pull request by sondreso**
**Issue**
Needed for https://github.com/equinor/ert/issues/763

### CODE REVIEW

File: lib/enkf/local_ministep.cpp [hunk 1]
```diff
@@ -162,6 +165,14 @@ void local_ministep_add_obsdata( local_ministep_type * ministep , local_obsdata_
   }
 }
 
+void local_ministep_add_obs_data( local_ministep_type * ministep , obs_data_type * obs_data) {
+  // if (ministep->obs_data != NULL){
+  //   obs_data_free(ministep->obs_data);
+  //   ministep->obs_data = NULL;
+  // }
```
Comment (sondreso) [inline lib/enkf/local_ministep.cpp]: TODO: Fix leak

File: python/res/enkf/local_ministep.py [hunk 2]
```diff
@@ -58,6 +59,10 @@ def getName(self):
         """ @rtype: str """
         return self.name()
 
+    def getObsData(self):
```
Comment (ManInFez) [inline python/res/enkf/local_ministep.py]: use snake_eyes?

File: lib/enkf/obs_data.cpp [hunk 3]
```diff
@@ -190,6 +190,10 @@ active_type obs_block_iget_active_mode( const obs_block_type * obs_block , int i
 }
 
 
+bool obs_block_iget_is_active( const obs_block_type * obs_block , int iobs) {
+  return obs_block->active_mode[ iobs ] == ACTIVE;
```
Comment (ManInFez) [inline lib/enkf/obs_data.cpp]: whitespace in brackets

**Comment by ManInFez** [comment]
Maybe a test would be nice?

File: python/res/enkf/local_ministep.py (hunk 2 above)
Comment (sondreso) [reply python/res/enkf/local_ministep.py]: Agreed, but do we have any "official" guidelines for what we do when adding new features to files with wrong formatting?

**Comment by ManInFez** [comment]
Test this please!

**Comment by ManInFez** [comment]
Test this please!

**Comment by sondreso** [comment]
Test this please!

**Review by ManInFez — APPROVED** [review]
Looks good! jenkins komodo test is failing due to jenkins being troublesome...

**Comment by ManInFez** [comment]
remember to create a new tag with bumped minor version :)

**Pull request merged**